scripts/_secrets.sh

```diff
@@ -58,7 +58,8 @@ function _delete_all_secrets() {
                 "uhd-${env}-esri-maps-service-credentials"
                 "uhd-${env}-slack-webhook-url"
                 "uhd-${env}-cognito-service-credentials"
-                "uhd-${env}-auth-secret")
+                "uhd-${env}-auth-secret"
+                "uhd-${env}-revalidate-secret")
```

Comment:
This makes sure that the teardown process will clean up and delete this new secret when the environment is being torn down, otherwise AWS will simply mark it for deletion and schedule it for deletion after a default period of time, which is 30 days